File: JavaScript/01.JavaScript-Basic-February-2021/14.exams/2019.05.02_03-Exam/6.vetParking.js
function vetParking(input) {
    let days = Number(input[0]);
    let hours = Number(input[1]);

    let price = 0;
    let total = 0;
    for (let i = 1; i <= days; i++) {
        let dayTotal = 0;
        for (let j = 1; j <= hours; j++) {
            if (i % 2 == 0 && j % 2 == 1) {
                price = 2.5;
            } else if (i % 2 == 1 && j % 2 == 0) {
                price = 1.25;
            } else {
                price = 1;
            }
            dayTotal += price;
        }
        console.log(`Day: ${i} - ${dayTotal.toFixed(2)} leva`);
        total += dayTotal;
    }
    console.log(`Total: ${total.toFixed(2)} leva`);
}

//vetParking(["2", "5"]);
vetParking(["5", "2"]);

File: Python/01.Python-Basic-April-2019/15.exams/exam_2019.05.02/6.vet_parking.py
days = int(input())
hours = int(input())

price = 0
total = 0
for i in range(1, days + 1):
    total_day = 0
    for j in range(1, hours + 1):
        if i % 2 == 0 and j % 2 == 1:
            price = 2.50
        elif i % 2 == 1 and j % 2 == 0:
            price = 1.25
        else:
            price = 1

        total_day += price
    print(f'Day: {i} - {total_day:.2f} leva')

    total += total_day

print(f'Total: {total:.2f} leva')